Makefile

```diff
@@ -223,16 +224,25 @@ bundle: manifests kustomize ## Generate bundle manifests and metadata, then vali
 	cp dependencies.yaml ./bundle/metadata
 	operator-sdk bundle validate ./bundle
 	/bin/bash hack/pin-custom-bundle-dockerfile.sh
+	sed -i custom-bundle.Dockerfile.pinned -e "s|quay.io/openstack-k8s-operators/openstack-operator-storage-bundle.*|$(BUNDLE_STORAGE_IMG)|"
+
 
 .PHONY: bundle-build
 bundle-build: bundle ## Build the bundle image.
 	podman build -f custom-bundle.Dockerfile.pinned -t $(BUNDLE_IMG) .
 
-
 .PHONY: bundle-push
 bundle-push: ## Push the bundle image.
 	$(MAKE) docker-push IMG=$(BUNDLE_IMG)
 
+# NOTE: when building bundles locally correct make order is: bundle, dep-bundle-build-push,
+# bundle-build, bundle-push
+dep-bundle-build-push: bundle ## Build and push dependency bundle images (storage, etc)
```

Comment:
can we add the note here that it is shown when running `make help`?

Reply:
Sure. I moved it inline with the main comment so it should show up.